helm-charts/common/common-library/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+name: common-library
+description: A Helm chart for the common templates
+version: 1.2.0
```

Comment:
Please use '0-latest' as the default version. In each release, we'll replace the version value automatically by CI.

Reply:
Please include this rationale in the commit message